### ISSUE TITLE
Different accuracy after model conversion from keras to caffe

### ISSUE THREAD

**Issue by salwaelk** [microsoft/MMdnn#823]
I converted a model from tf.keras to caffe, the model with caffe gives higher performance in terms of accuracy, and for the same image, the inference confidence if higher with caffe than with tf.keras. why is that?

**Comment by linmajia** [comment]
@salwaelk , thank you very much for the feedback. Do you mean that the target Caffe model has better learning performance (accuracy) than the original tf.keras model? If so, I think it is because the different internal implementation of corresponding operators. And I suggest that you try more images to test the overall learning performance (accuracy).

**Comment by salwaelk** [comment]
@limajia Thanks for your answer. Yes, the target Caffe model has a higher accuracy than the original tf.keras model. I understand that the implementations are different, but is it still the case even when cafe and tensorflow are both using the same versions of installed acceleration libraries such as cuda and cudnn?

**Comment by linmajia** [comment]
@salwaelk , e.g., the execution ordering of the same CUDA/cuDNN APIs may cause floating-point result a little different.
